File: VB/TemplateCardCaptionLocalization/MyGridControl.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraGrid
Imports DevExpress.XtraGrid.Registrator

Namespace TemplateCardCaptionLocalization
	Public Class MyGridControl
		Inherits GridControl
		Protected Overrides Sub RegisterAvailableViewsCore(ByVal collection As InfoCollection)
			MyBase.RegisterAvailableViewsCore(collection)
			collection.Add(New MyLayoutViewInfoRegistrator())
		End Sub
	End Class
End Namespace

File: CS/TemplateCardCaptionLocalization/MyGridControl.cs
using DevExpress.XtraGrid;
using DevExpress.XtraGrid.Registrator;

namespace TemplateCardCaptionLocalization {
    public class MyGridControl :GridControl {
        protected override void RegisterAvailableViewsCore(InfoCollection collection) {
            base.RegisterAvailableViewsCore(collection);
            collection.Add(new MyLayoutViewInfoRegistrator());
        }
    }
}